File: JAVA/S1_BASIC/L5_BASICRECURSION/print.java

class recursion{
    void print(int n){
        if(n==0)
        return;
        System.out.print("hello"+" ");
        print(n-1);
    }
}
public class print{
    public static void main(String[]args){
        recursion r=new recursion();
        r.print(5);

    }
}

File: python/S1_BASIC/L5_BASICRECURSION/print.py
class Recursion:
    def __init__(self,n):
        self.n=n
    def Print(self,n):
        if n==0:
            return
        print("hello",end="\n")
        self.Print(n-1)
if __name__=="__main__":
    n=int(input())
    R=Recursion(n)
    R.Print(n)